bundles/org.pgcase.xobot.dbproc.jdbc/src/org/pgcase/xobot/dbproc/jdbc/DbprocJdbc.java

```diff
@@ -1,7 +1,51 @@
 package org.pgcase.xobot.dbproc.jdbc;
 
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import org.pgcase.xobot.dbproc.jdbc.functions.JdbcFunctionExtractor;
+
 public class DbprocJdbc {
```

Comment:
Please rework to remove code duplications

Reply:
Done